LEAF_Request_Portal/templates/subindicators.tpl

```diff
@@ -116,7 +116,13 @@
                         return !valid;
                     },
                     setRequiredError: function() {
+                        $([document.documentElement, document.body]).animate({
+                            scrollTop: $('#<!--{$indicator.indicatorID|strip_tags}-->_required').offset().top
+                        }, 700);
                         $('#<!--{$indicator.indicatorID|strip_tags}-->_required').css({"background-color": "red", "color": "white", "padding": "4px", "font-weight": "bold"});
+                    },
+                    setRequiredOk: function() {
+                        $('#<!--{$indicator.indicatorID|strip_tags}-->_required').css({"background-color": "white", "color": "red", "padding": "0", "font-weight": "normal"});
```

Comment:
The same for the .css values, please.  Separate json variable.

Reply:
👍 Changed the css to an actual class on the style.css and implemented the style to toggle.